app/models/community_user.rb

```diff
@@ -24,6 +24,11 @@ def suspended?
     false
   end
 
+  # All undeleted posts on this community by this user.
+  def post_count
+    Post.unscoped.where(community_id: community_id).where(user: user).undeleted.count
```

Comment:
This is going to be expensive when run for multiple users and/or communities on one page. We're _probably_ alright at current scale, but we should ultimately cache this on the CommunityUser model for each community.

Reply:
It occurs to me that the community-specific post count shows up other places, including on the profile, where we call `@user.posts.undeleted.count`.  But I'm not quite sure how to do that for a non-local user -- any advice?  We're calling this from a user on site A and asking about the user on site B, C, D... .

Reply:
Filed https://github.com/codidact/qpixel/issues/1411 after discussion with @ArtOfCode- .